tools/rostopic/src/rostopic/__init__.py

```diff
@@ -1532,19 +1530,15 @@ def _rostopic_cmd_bw(argv=sys.argv):
     from optparse import OptionParser
     parser = OptionParser(usage="usage: %prog bw /topic", prog=NAME)
     parser.add_option("-w", "--window",
-                      dest="window_size", default=None,
+                      dest="window_size", default=-1,
```

Comment:
It's not required but I did this for simplification of the following logic for converting `window_size` to int since `int(None)` is not allowed. I can check for None there instead if that seems better.